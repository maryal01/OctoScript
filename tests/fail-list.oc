list<int> x = [1, "two"];