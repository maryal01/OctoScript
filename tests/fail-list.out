Fatal error: exception Failure("illegal List literal int expected in [1, "two"]")